chainer/functions/activation/lstm.py

```diff
@@ -220,9 +228,24 @@ def lstm(c_prev, x):
         >>> x = model.w(y) + model.v(h)
         >>> c, h = F.lstm(c, x)
 
-        It corresponds to calculate the input sources :math:`a, i, f, o` from
-        the current input ``y`` and the previous output ``h``. Different
-        parameters are used for different kind of input sources.
+        It corresponds to calculate the input array ``x``, or the input
+        sources :math:`a, i, f, o` from the current incoming signal ``y`` and
+        the previous outgoing signal ``h``. Different parameters are used for
+        different kind of input sources.
+
+    .. note::
+
+        We use the naming rule below.
+
+        - incoming signal
+            The formal input of the formulation of LSTM (e.g. in NLP, word
+            vector or output of previous RNN). The input of
```

Comment:
"output of previous RNN" is little confusing and seems to represent "previous h of the unit itself". Could you modify it to "output of lower RNN layer" or something like that?